samples/bluetooth/mesh/light/README.rst

```diff
@@ -43,10 +43,10 @@ The configuration overlay :file:`overlay-dfu.conf` enables DFU support in the ap
 
 * nrf52833dk_nrf52833
 
-While this overlay configuration is only applicable for the nRF52 Series in this sample, DFU over SMP can be utilized on other platforms as well.
+While this overlay configuration is only applicable for the nRF52 Series in this sample, DFU over Bluetooth Low Energy can be utilized on other platforms as well.
```

Comment:
```suggestion
While this overlay configuration is only applicable for the nRF52 Series in this sample, DFU over Bluetooth® Low Energy can be utilized on other platforms as well.
```

Reply:
Trademark is only necessary on the first occurrence of Bluetooth within a page/file. And we have that covered here.